Dp 19120 global nav drupal changes

arthurbaghdas opened the pull request:
**Description:**
Added multiple version headers. Added a config form to allow people to choose between mixed and hamburger version headers.

**Jira:** (Skip unless you are MA staff)
https://jira.mass.gov/browse/DP-19120


**To Test:**
- [ ] Go to /admin/config/mass/header
- [ ] Add a relative url for the page to show mixed version header

arthurbaghdas committed:
DP-19120: header navigation changes

arthurbaghdas committed:
Template changes

arthurbaghdas committed:
Adding changelog

arthurbaghdas committed:
Adding default conf

arthurbaghdas committed:
phpcs issue fix

arthurbaghdas committed:
Merge branch 'develop' into DP-19120-global-nav-drupal-changes

tom-fleming commented:
@arthurbaghdas I think we should add a test for checking that saving the config page hides/shows the horizontal nav. Otherwise, LGTM!

ygannett committed:
adjust theming templates for additional changes done in Mayflower

ygannett committed:
Merge branch 'develop' into DP-19120-global-nav-drupal-changes

ygannett committed:
DP-19328 Hamburgermenu for binder page (#406)

* update the <head> content generation logic in /themes/custom/mass_theme/templates/layout/page--node--without-main.html.twig for binder page

* add changelog for DP-19328

* remove commented out code

ygannett committed:
update the menu label change in Mayflower

tom-fleming committed:
Merge remote-tracking branch 'origin/develop' into DP-19120-global-nav-drupal-changes

tom-fleming committed:
Merge remote-tracking branch 'origin/develop' into DP-19120-global-nav-drupal-changes

ygannett committed:
match MF change in theming

ygannett committed:
Merge branch 'DP-19120-global-nav-drupal-changes' of https://github.com/massgov/openmass into DP-19120-global-nav-drupal-changes

arthurbaghdas committed:
DP-19876: Contextual login links are missing in branch with new global nav (#468)

* DP-19876: Contextual links fix

* DP-19876: Added changelog

Co-authored-by: Tom Fleming <tom@lastcallmedia.com>

tom-fleming committed:
Merge remote-tracking branch 'origin/develop' into DP-19120-global-nav-drupal-changes

ygannett committed:
DP-20203 Missing alert overlay in info details page (#490)

* add alert overlay container to page--node--without-main.html.twig

* add changelog for DP-20203

tom-fleming committed:
Added back header_mixed_urls setting - DP-19120

tom-fleming committed:
Resolve DP-19120 merge conflicts (#492)

* config change and changelog (#479)

* DP 20094 help text change for data resource type field (#480)

* help text changes

* changelog

* simple text change to trigger rebuild

* changelog update and remove old changelog files

* Add data fields to service_page (#474)

* Add data fields to service_page

* Use related terms

* Add support for service_page in mass_related_terms token.

* add field to be used for description

* conditional logic and form layout changes

* Fix coding standards.

Co-authored-by: Joe Galluccio <joseph.galluccio@state.ma.us>

* Remove balancer in Terraform (#481)

* Remove balancer IP address.

* Add changelog

* try uppercase

* Declare the variable again. Add logging

* Remove super verbose logging.

* Upper

* Dp 20172 promo button text size (#486)

* config changes

* changelog

* Add changelog (#485)

* DP-19747 - Reduce log noise (#484)

* Remove debug loggings.

* Add changelog

* CS

* Fix doc comment.

* bool

* Update basic iframe URL so that node ID is correctly passed to Supers… (#483)

* Update basic iframe URL so that node ID is correctly passed to Superset filter.

* changelog.

* fixing syntax on changelog

Co-authored-by: joeg8612 <joseph.galluccio@state.ma.us>

* DP-19787: Fix the needs attention views (#476)

* DP-19787: Fix the needs attention views

The calls to superset were calling a non-existant old view.
In addition, the superset API has changed and this decreases
the batch size from 1000 to 500 to eliminate timeouts.

* DP-19787: Add changelog

* DP-19787: Increase the cron timeout for superset queue items

* DP-19787: Restart mass_superset queue

* Added changelog - DP-19120

Co-authored-by: joeg8612 <joseph.galluccio@state.ma.us>
Co-authored-by: CircleCI Deployment Bot <circleci@example.com>
Co-authored-by: Moshe Weitzman <weitzman@tejasa.com>
Co-authored-by: Yukiko Gannett <yukikogannett@gmail.com>
Co-authored-by: adamcogbill <33554893+adamcogbill@users.noreply.github.com>
Co-authored-by: Robert Stephens <roberttstephens@gmail.com>

tom-fleming committed:
Resolved merge conflicts and changes from develop - DP-19120

ygannett reviewed (approved):
LGTM.

tom-fleming commented:
🎉
